splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/CheckBoxComponent.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Switch from '@splunk/react-ui/Switch';
+
+class CheckBoxComponent extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+    handleChange = (e, {value}) => {
+        this.props.handleChange(this.props.id, value);
+    };
+
+    render() {
+        return (
+            <Switch
+                key={this.props.field}
+                value={this.props.value}
+                onClick={this.handleChange}
+                selected={this.props.value === 1 ? true : false}
```

Comment:
@dkhatri-crest What should be the value of **this.props.value**?
If it's only 0 or 1 then we can simplify our statement like: **this.props.value ? true : false**

Reply:
Updated